sdks/python/apache_beam/io/gcp/bigquery.py

```diff
@@ -1570,6 +1584,10 @@ class _ReadFromBigQuery(PTransform):
       bucket where the extracted table should be written as a string or
       a :class:`~apache_beam.options.value_provider.ValueProvider`. If
       :data:`None`, then the temp_location parameter is used.
+    backwards_compatible_data_format (bool): By default, this transform reads
+      data in native Python types that come from Avro-exports of BigQuery. By
+      setting this flag to True, the transform will return JSON-types, like
+      the older BigQuerySource.
```

Comment:
1. Do all BQ types have a native python type?
2. What are JSON-types?

Reply:
I've added a link to https://cloud.google.com/bigquery/docs/loading-data-cloud-storage-avro#avro_conversions for details on the conversions. I've also rephrased the doc. LMK what you think.